codeworld-base/src/Extras/Colors.hs

```diff
@@ -0,0 +1,271 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE PackageImports    #-}
+{-# LANGUAGE RebindableSyntax #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ParallelListComp #-}
+{-# LANGUAGE RecordWildCards #-}
+
+-- | Convenience functions for specifying colors by name, hexadecimal value,
+-- integer value between 0 and 255, and a few other ways.
+module Extras.Colors(
+    -- $intro
+    painted, colorNamed, rgb, greyed, withAlpha
+    ) where
+
+import Prelude
+
+-------------------------------------------------------------------------------
+-- $intro
+-- = Extra Colors
+--
+-- To use the extra features in this module, you must begin your code with this
+-- line:
+--
+-- > import Extras.Colors
+--
+
+-- | The given picture painted with the color corresponding to the given
+-- color name, which must be a valid argument of the 'colorNamed' function.
+painted :: (Picture,Text) -> Picture
+painted(pic,name) = colored(pic,colorNamed(name))
+
+-- | This function allows you to specify color components in the range 0 to 255
+-- instead of 0 to 1.
+rgb :: (Number,Number,Number) -> Color
+rgb(r,g,b) = RGB(r/256,g/256,b/256) -- Not exact, but colors aren't anyway...
```

Comment:
Shouldn't this be dividing by 255 instead of 256?

Reply:
Yes. It should be 255. That's what the *not exact* comment in the code was about.  I will correct it.

I just thought: 255 is the worst possible divisor, while 256 is the best, and there is very little visible difference...